src/switch-angular-builder.ts

```diff
@@ -0,0 +1,33 @@
+import {versions} from './angular-versions';
+import {AngularJSON} from './types/angular-json';
+
+const oldBuilder = '@angular-devkit/build-ng-packagr:build';
+const newBuilder = '@angular-devkit/build-angular:ng-packagr';
+
+export function switchAngularBuilder(
```

Comment:
suggestion: Use *replace* rather than *switch* in the name. Also make it more specific, for example `replaceLibraryBuilder`.